main.go

```diff
@@ -32,6 +32,7 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
+	metallbiov1alpha1 "github.com/metallb/metallb-operator/api/v1alpha1"
```

Comment:
already imported as metallbv1alpha1

Reply:
same here this also outdated that has been removed in the latest diffs